ultra/ultra/evaluate.py

```diff
@@ -60,7 +60,7 @@ def evaluation_check(
     agent_ids_to_evaluate = [
         agent_id
         for agent_id in agent_ids
-        if episode.index % eval_rate == 0
+        if (episode.index + 1) % eval_rate == 0
```

Comment:
Yes, I think this was added because I was kind of in the mindset of curriculum learning. And I do not evaluate in the first episode. I can revert that back